docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -268,12 +265,12 @@ test('Working With iframe test', async t => {
 
 ## Handle Native Dialogs
 
-TestCafe allows you to handle native browser dialogs that are invoked during the test run.
+TestCafe allows you to handle native dialogs that the browser may display during the test run.
 
 You can close [alert](https://developer.mozilla.org/en-US/docs/Web/API/Window/alert) and
 [beforeunload](https://developer.mozilla.org/en-US/docs/Web/Events/beforeunload) dialogs,
 choose an option in [confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm) dialogs
-or provide text for [prompt](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) dialogs.
+or hand over text for [prompt](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) dialogs.
```

Comment:
`or hand over text for...`

I propose using the most simple verbs because most of our customers are not native speakers.
`provide` was a good word.

Reply:
`provide` is banned by correctors currently

Reply:
`provide` is currently banned by correctors

Reply:
> `provide` is currently banned by correctors

I guess we could use `supply`

Reply:
Ok. Let it be so.